server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/UploadContract.scala

```diff
@@ -138,13 +136,33 @@ trait UploadContract {
     .when
       .post(s"/upload/$ACCOUNT_ID/")
     .`then`
-      .statusCode(SC_OK)
+      .statusCode(SC_BAD_REQUEST)
+      .extract
+      .body
+      .asString
+
+    assertThat(response.contains("Attempt to upload exceed max size"))
+      .isTrue
+  }
+
+  @Test
+  def shouldRejectWhenUploadFilesExceedMaxSize(): Unit = {
+    val response: String = `given`
+      .basePath("")
+      .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .contentType(ContentType.BINARY)
+      .body(VALID_INPUT_STREAM)
+      .body(VALID_INPUT_STREAM)
```

Comment:
I don't see the added value of shouldRejectWhenUploadFilesExceedMaxSize test.

Reply:
Then it removing